tiling_segmentation.py

```diff
@@ -184,55 +186,263 @@ def make_dataset_file_name(exp_name: str, min_annot: Number, dataset: str, attr_
         dataset_file_name = f'{exp_name}{sampling_str}_{dataset}.csv'
         return dataset_file_name, sampling_str
 
+    @staticmethod
+    def _save_vrt_read(aoi):
```

Comment:
This function is very similar to create_new_raster_from_base() (in utils/). Can you check if the existing utility can be used? It can certainly be improved if you feel like getting into that.

Reply:
yeah, I tried to apply your function create_new_raster_from_base() as you know, but at some point PermissionError happened when I used it. Maybe it is because I use Windows, don't know, but _save_vrt_read() works well. I'll look into how the existing function can be improved so it works for different file systems.